src/main/java/net/mcreator/minecraft/ElementUtil.java

```diff
@@ -34,6 +34,15 @@
 
 public class ElementUtil {
 
+	/**
+	 * Provides a predicate to check the type of data list entries
+	 * @param type The type that the entry has to match
+	 * @return A predicate that checks if the type matches the parameter
+	 */
+	public static Predicate<DataListEntry> typeMatches(String type) {
```

Comment:
You could also apply this predicate to gamerules in this file, so all elements use the same system